reference/api/dump.md

```diff
@@ -28,35 +28,12 @@ The task queue will not process any further tasks during dump creation, but you
 
 ```json
 {
-  "uid": "20200929-114144097",
-  "status": "in_progress",
-  "startedAt": "2020-09-29T11:41:44.392327Z"
+  "taskUid": 0,
+  "indexUid": null,
```

Comment:
We should either add a note here or add this to the 'Understanding tasks"  and "Summarized `task` Object for `202 Accepted`" tables in `learn/advanced/asynchronous_operations.md` that the `indexUid` will be `null` when task type is `dumpCreation`